buildenv/jenkins/common/pipeline-functions.groovy

```diff
@@ -661,6 +661,18 @@ def retry_and_delay(command, numRetries = 3, waitTime = 60, units = 'SECONDS') {
 }
 
 def setup_pull_request() {
+   def PARSED_MUTIPLE_COMMENT = params.ghprbCommentBody.toLowerCase().split("\\r?\\n")
+   for (COMMENT in PARSED_MUTIPLE_COMMENT) {
+       def comment = COMMENT.toLowerCase().tokenize(' ')
+       if ((comment[0] == "jenkins") && ((comment[1] == "compile") || (comment[1] == "test"))){
+           setup_pull_request_single_comment(comment)
+           return                                                               
```

Comment:
extra space after `return`

Reply:
Hey @AdamBrousseau! Sorry I accidentally changed the code. It is reverted now. Thanks for testing it again.